Library/CMakeLists.txt

```diff
@@ -37,8 +39,8 @@ set_property(TARGET bimg PROPERTY FOLDER Dependencies/bgfx)
 set_property(TARGET bx PROPERTY FOLDER Dependencies/bgfx)
 
 set(CMAKE_USE_WINSSL ON CACHE BOOL "Set cURL to use WinSSL by default.")
-add_subdirectory(Dependencies/curl EXCLUDE_FROM_ALL)
-set_property(TARGET libcurl PROPERTY FOLDER Dependencies/curl)
+#add_subdirectory(Dependencies/curl EXCLUDE_FROM_ALL)
+#set_property(TARGET libcurl PROPERTY FOLDER Dependencies/curl)
```

Comment:
Is curl part of the Android OS? It looks like it might not be, which means we'll have to get our curl build to work for Android.

Reply:
We will have to! I forgot to update the description for that. It doesn't seem to be easily buildable with cmake but it might also be just some configuration that I'm missing.